docs/guide/database-insert-update.md

```diff
@@ -78,7 +92,7 @@ In the example above, `self` refers to the `dog` piece type module since that is
 
 Updating pages works the same way as pieces with the same arguments to the `update` method. There are a few things to keep in mind when working on pages, however.
 
-**We often call the page `update` method from `self.apos.page`.** `self.update` does work from individual page type modules and sometimes that is the right way to call it. But since pages can change their `type` property (unlike pieces) they share a single `update` method from the `@apostrophecms/page` module. `self.update` on individual page type modules is simply a wrapper that method.
+**We often call the page `update` method from `self.apos.page`.** `self.update` does work from individual page type modules and sometimes that is the right way to call it. But since pages can change their `type` property (unlike pieces) they share a single `update` method from the `@apostrophecms/page` module. `self.update` on individual page type modules is simply a wrapper around that method.
```

Comment:
I would just skip the thing about calling it from individual page type modules instead. It's never "wrong" to call it from the page module and it's potentially wrong not to.